### DISCUSSION TITLE
De-clutter task view by collapsing repeating tasks

### DISCUSSION THREAD

**Discussion by gdt** [TechbeeAT/jtxBoard#1193] [Ideas]
I am new to VTASK entirely, and to jtxBoard.

I would like to create a recurring task, to happen every day at the same time.  I would like an alarm if I have not checked it off for that day.  This seems to work.  But, when I look at at the all task view, i see all the future days.    I think this is a bug; this is one task with a daily reminder, not a bunch of separate things, and I want to see it only once.  Sure, if I open it I should see the repeating properties.   And perhaps if a calendar is displaying tasks it should appear.  But on the apps tast list, this keeps me from seeing anything else.

I realize I can perhaps add a tag and filter that out (but there isn't "exclude tasks with the daily tag", and I don't really want to exclude, but instead just have the next one appear).

If I'm missing something, I'd appreciate a clue.  And sorry if I didn't find a matching discussion by keyword searching.

**Comment by gdt** [comment]
I found that the menu in the "search tasks" box is about display options, not search, and that it has an option to limit recurring to one item.  So I amend my suggestion to
  - make show-once be default
  - move the triple-dot menu out of the text entry, because it's not about configuring the behavior of what happens when you enter, but large scope.

**Comment by patrickunterwegs** [comment]
Hi @gdt 
thanks for your remarks!

make show-once be default -> I think it's better understandable for all users when the option is deactivated by default. At least users who are bothered by the many copies would find the option and set it consciously whereas users that expect the copies might think it's a bug when the recurrences are missing.

move the triple-dot menu out of the text entry, because it's not about configuring the behavior of what happens when you enter, but large scope. -> I have checked other apps and it seems to be pretty intuitive for most users. Besides putting the three dots there outside of the field just looks ugly... I just tried it 😉
